### PR TITLE
Fix security warning.

### PR DESCRIPTION
**Pull request by yaohaizh**
Fix security alerts on the github repo main page.

### CODE REVIEW

File: build-tools/src/main/resources/checkstyle.xml [hunk 1]
```diff
@@ -84,11 +80,17 @@
...
+            <property name="commentFormat" value="CHECKSTYLE IGNORE (\w+) FOR NEXT (\d+) LINE"/>
+            <property name="checkFormat" value="$1"/>
+            <property name="influenceFormat" value="$2"/>
+        </module>
         <!-- Checks for Javadoc comments.                     -->
         <!-- See http://checkstyle.sf.net/config_javadoc.html -->
 
         <module name="JavadocMethod">
             <property name="scope" value="public"/>
+            <property name="allowMissingParamTags" value="true"/>
```
Comment (jianghaolu) [inline build-tools/src/main/resources/checkstyle.xml]: Why is this needed?

File: build-tools/src/main/resources/checkstyle.xml [hunk 2]
```diff
@@ -173,8 +175,6 @@
         <!-- See http://checkstyle.sf.net/config_modifiers.html -->
         <module name="ModifierOrder"/>
         <module name="RedundantModifier"/>
-        <module name="FileContentsHolder"/>
```
Comment (jianghaolu) [inline build-tools/src/main/resources/checkstyle.xml]: Why is this removed? From the docs it seems this is needed if used in conjunction with `SuppressWithNearbyCommentFilter`: https://checkstyle.sourceforge.io/version/6.18/config_filters.html#SuppressWithNearbyCommentFilter

>  Usage: This filter only works in conjunction with a FileContentsHolder, since that check makes the suppression comments in the Java files available. A configuration that includes this filter must configure FileContentsHolder as a child module of TreeWalker.

Comment (yaohaizh) [reply build-tools/src/main/resources/checkstyle.xml]: According to this release note: https://checkstyle.sourceforge.io/releasenotes.html#Release_8.2, this is not necessary again after version 8.2. Here we use 8.24 to make it consist with the latest setup on the azure-sdk-for-java-repo.

File: build-tools/src/main/resources/checkstyle.xml (hunk 1 above)
Comment (yaohaizh) [reply build-tools/src/main/resources/checkstyle.xml]: There are too many warnings about violation of this rule. Given this, I suppress this temporarily. If the security work is not urgent, I would like to fix the issue on generator and then on the fluent code base.
